client/templates/dashboard/dashboard.js

```diff
@@ -1,3 +1,7 @@
+Template.dashboardShopifyProducts.onCreated(function () {
+  this.subscribe('ShopifyProducts/CurrentImport');
+});
+
```

Comment:
Also, if this was a customer facing subscription, we can use Template.subscriptionsReady to conditionally display a loading template if the template subscriptions are not ready yet, so we get that functionality built in as well.